Monika After Story/game/script-food.rpy

```diff
@@ -0,0 +1,165 @@
+    
+label mas_food_water:
+    m 1eua "I'm really happy to hear that, [player]!"
+    m 1eub "Water is probably the healthiest thing that you could drink."
+    m 3eua "Seeing how we can only live for a few days without it."
+    m 3eud "Just promise you get plenty to drink each day."
+    m 2lud "I don't want you getting sick because you didn't drink any."
+    m 2eua "Even if it's just a glass or two, do it for me."
+    return
+    
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_milk',prompt='Milk',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+    
+label mas_food_milk:
+    m 1eub "You know, I've never really drank a lot milk before."
+    m 3eub "It wasn't because I didn't like it either."
+    m 3eua "I just never really thought about it, unless I put it in coffee, or mixed chocolate in with it."
+    m 1hub "I guess I could learn to love it if you do, [player]."
+    m 1hua "Be sure to pour an extra glass for me next time."
+    return
+    
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_hotchocolate',prompt='Hot Chocolate',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+    
+label mas_food_hotchocolate:
+    if mas_isWinter():
+        m 1hub "That's so nice, [player]!"
+        m 4eub "There's really nothing quite like the comfort of hot chocolate when it's chilly outside, is there?"
+        m 1eub "In winter I always enjoy wrapping up in some warm clothes and unwinding with a cup myself."
+        m 1eua "It's a great way to relax."
+        m 1kua "Maybe some day when it's cold out, we could sit back and have a mug together."
+        m 1duu "That would be a dream come true, my love!"
+        
+    else:
+        m 1hua "That sounds really good right about now."
+        m 1hub "There's nothing better than a nice glass of hot cocoa after a long day."
+        m 1duu "I can't wait until we can drink some together."
+    return
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_chocolatemilk',prompt='Chocolate Milk',category=[store.mas_food.TYPE_DRINK],unlocked=True,),code='FOO')
+    
+label mas_food_chocolatemilk:
+    m 3eub "That's wonderful, [player]."
+    m 3eub "Although I never drank regular milk too often, there was something different when you mixed chocolate with it."
+    m 1lud "..."
+    m 1rusdlb "How much do you have [player]?"
+    m 1hua "Mind pouring me a glass? ehehe~"
+    return
+    
+init 5 python:
+    addEvent(Event(persistent._mas_food_database,'mas_food_pizza',prompt='Pizza',category=[store.mas_food.TYPE_FOOD],unlocked=True,),code='FOO')
+    
+label mas_food_pizza:
+    m 1hua "Pizza is such a great treat!"
+    m 3eua "It's usually not the healthiest food, of course, but I think on occasion, it's fine to just treat yourself, you know?"
+    m 3hub "I think it's really interesting that pizza was created almost by accident."
```

Comment:
I guess there isn't really anything wrong with this expression, but I'm not sure if this fits Monika while she's telling a fun fact that isn't really romantic or really jokeful in any way.